scripts/SANS/SANSUtility.py

```diff
@@ -1025,6 +1025,60 @@ def create_add_algorithm(self, is_overlay):
             return PlusWorkspaces()
 
 
+def _clean_logs(ws, estimate_logs):
+    """
+    Remove bad proton charge times from the logs, if present.
+    :param ws: The workspace to clean
+    :param estimate_logs: bool. If true, estimate "good" values for bad proton logs
+    """
+    run = ws.getRun()
+    if run.hasProperty("proton_charge"):
+        pc = run.getProperty("proton_charge")
+        first_log_time = pc.firstTime().toISO8601String()
```

Comment:
So this would be better to create a time of 1990 and then compare the date times rather than convert to string and parse out the date.